style.css

```diff
@@ -0,0 +1,181 @@
+@import url("https://fonts.googleapis.com/css?family=Poppins:200i,400&display=swap");
+
+:root {
+  --color-white: #f3f3f3;
+  --color-darkblue: #4444a8;
+  --color-darkblue-alpha: rgba(34, 61, 71, 0.8);
+  --color-green: #d59cf7;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Poppins", sans-serif;
+  font-size: 1rem;
+  font-weight: 400;
+  line-height: 1.4;
+  color: var(--color-white);
+  margin: 0;
+}
+
+/* mobile friendly alternative to using background-attachment: fixed */
```

Comment:
Please remove the comment on line 25 to make your code look more professional.😊😊